src/components/errors/errors.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+type Props = {
+  errorMessage: string;
+  onClose: (message: string) => void;
+};
+
+export const Errors: React.FC<Props> = ({ errorMessage, onClose }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={`notification is-danger is-light has-text-weight-normal ${!errorMessage && 'hidden'}`}
```

Comment:
Classnames here and in all other places - https://www.npmjs.com/package/classnames